step_start_instance.go

```diff
@@ -33,9 +33,6 @@ func (s *stepStartInstance) Run(state multistep.StateBag) multistep.StepAction {
 
 	instance, err := s.provider.Start(ctx, buildJob.StartAttributes())
 	if err != nil {
-		logger.WithField("err", err).Error("couldn't start instance")
```

Comment:
What's the reasoning behind moving this log entry down? Won't this error get swallowed if we return at line 49 (`return multistep.ActionHalt`) ?

Reply:
The error condition handled in that block is specifically for `workererrors.JobAbortError`, which is not an internal error that should necessarily be logged as such.  Moving this code also makes it more consistent with other places where we run `buildJob.Requeue`.